stats_prometheus.go

```diff
@@ -34,16 +34,20 @@ func getOutputNewCounterVec() *prometheus.CounterVec {
 	)
 }
 
-func getFalcoNewCounterVec() *prometheus.CounterVec {
+func getFalcoNewCounterVec(config *types.Configuration) *prometheus.CounterVec {
+	labelnames := []string{
+		"rule",
+		"priority",
+		"k8s_ns_name",
+		"k8s_pod_name",
+	}
+	for key := range config.Customfields {
+		labelnames = append(labelnames, key)
+	}
```

Comment:
Can you check which characters are allowed? I don't remember if `-` are allowed, this is why we use `_`